docs/usage.md

```diff
@@ -82,10 +82,17 @@ You can reset a scenario by using any of the following methods.
 
     ![Delete Scenario Records from Scenario Record](res/delete-from-scenario-record.png)
 
-### Guidelines to Create Scenario Playbook
+### Guidelines to Create a Scenario Playbook
 
-- The Scenario playbooks nomenclature should be "Scenaio - <playbook_name>"
-- A `Scenario` tag should be given to the playbook
-- The playbooks should conclude with a 'Set Variable' step, where the variable named `recordIRIs` is assigned a value representing the list of all demo records created by the scenario playbook.
+These guidelines are drafted to address validations implemented in FortiSOAR&trade; 7.4.3 and later.
+
+1. **Scenario Playbook Naming Convention:**
+   - The naming format for scenario playbooks should follow this structure: `Scenario - <playbook_name>`. Replace `<playbook_name>` with an appropriate name that reflects the purpose of the playbook.
+
+2. **Scenario Tag Assignment:**
+   - Each scenario playbook should be tagged with the label `Scenario`. This tag helps organize and identify playbooks specifically designed for scenarios.
+
+3. ***Set Variable* Step for recordIRIs:**
+   - Conclude each scenario playbook with a *Set Variable* step. In this step, create a variable named `recordIRIs` and assign it a value representing the list of all demo records generated by the scenario playbook.
```

Comment:
Please add note suggesting that going forward you should always create a scenario playbook using `recordIRIs` as a output variable instead of "@id"